### ISSUE TITLE
docs: add installation instructions for configureStore

### ISSUE THREAD

**Issue by diracs-delta** [zalmoxisus/redux-devtools-extension#776]
See corresponding redux docs: https://redux-toolkit.js.org/api/configureStore

In the latest CRA template for `redux-typescript`, the root store is configured via `configureStore` instead of `createStore`. The installation instructions don't mention anything for this case.

With `configureStore`, enabling devtools is simple as appending the following line to the setup:

```
export const store = configureStore({
  ...
  devTools: true
  ...
})
```

We can save users a lot of future headache by adding this to the README.

**Comment by nguyenduckhai** [comment]
Do you have any problem with devTools on Chrome? Even though I have set devtools in ConfigureStore to True, DevTools on chrome can not detect my redux state.

**Comment by steve-taylor** [comment]
Yep, it simply does not work. Another completely broken tool to throw on the massive pile of completely broken frontend tools. These tools often work by themselves, but as soon as you combine them, it ends in tears.
